src/core/generateZodSchema.test.ts

```diff
@@ -1587,13 +1590,14 @@ describe("generateZodSchema", () => {
     /**
      * @discriminator id
      **/
-    export type A = { name: string; } | { id: "2"; age: number; }`;
+    export type A = { id: "1"; name: string; } | { id: "2"; age: number; }`;
```

Comment:
The test is called `should fall back to union when discriminator is missing`, adding `id: "1"` is testing something else